src/dataServices/dynamoDbBundleService.ts

```diff
@@ -37,16 +42,24 @@ export class DynamoDbBundleService implements Bundle {
 
     private dynamoDb: DynamoDB;
 
-    private maxExecutionTimeMs: number;
+    private readonly maxExecutionTimeMs: number;
 
     private static readonly dynamoDbMaxBatchSize = 25;
 
+    private readonly versionedLinks: any;
```

Comment:
Instead of any be more precise

```suggestion
    private readonly versionedLinks: Record<string, string[]>;
```